src/client/components/MegaMenu/MegaMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import { Heading, LinkList, MegaMenu } from '@amsterdam/design-system-react';
+import { ChapterMenuItem } from '../../../universal/config';
+import styles from './MegaMenu.module.scss';
+import { MenuItem } from '../MainHeader/MainNavBar.constants';
+
+type Props = {
+  menuItems: MenuItem[];
+  chapters: ChapterMenuItem[];
+};
+
+export default function Menu({ menuItems, chapters }: Props) {
+  return (
+    <MegaMenu>
+      <div className={styles.menu}>
+        <Heading level={3} size="level-3">
+          Thema’s
+        </Heading>
+
+        <Heading level={3} size="level-3">
+          Categorieën
+        </Heading>
```

Comment:
Deze volgorde is raar. Kan deze boven de volgende Categorie geplaatst worden?

Reply:
ik gebruik hier css grid. 
@import '../../styles/_global.scss';

.menu {
  @include centered-block;
  padding: 4rem 0 6rem;
  display: grid;
  grid-template-columns: 2fr 1fr;

  :global(.ams-icon){
    display: none;
  }

  & > :nth-child(3) :global(.ams-link-list__link) {
    font-weight: 800;
  }

  @include mq-phone() {
    grid-template-columns: 1fr;

    & > :nth-child(2) {
      order: 3;
    }

    & > :nth-child(4) {
      order: 4;
    }
  }

  & > div {
    padding-left: unset;
    padding-right: unset;
  }
}

Reply:
CSS = Styling, HTML = Semantische opmaak van het document. https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics